packages/@react-aria/table/src/useTableColumnResize.ts

```diff
@@ -63,14 +64,31 @@ export function useTableColumnResize<T>(props: AriaTableColumnResizeProps<T>, st
   let id = useId();
   let isResizing = useRef(false);
   let lastSize = useRef(null);
+  let editModeEnabled = state.tableState.isKeyboardNavigationDisabled;
+  let resizeOnFocus = !!triggerRef?.current;
```

Comment:
Reading from a ref in render. Can you move this into the event handlers where it is used?

Reply:
argh, thanks for catching, keep forgetting